packages/integrations/useAxios/index.ts

```diff
@@ -112,9 +142,18 @@ export function useAxios<T = any>(url: string, ...args: any[]): UseAxiosReturn<T
     isLoading.value = loading
     isFinished.value = !loading
   }
-  const execute = (config: AxiosRequestConfig = {}) => {
+  const execute: OverallUseAxiosReturn<T>['execute'] = (executeUrl: string | AxiosRequestConfig | undefined = url, config: AxiosRequestConfig = {}) => {
+    let _url = ''
```

Comment:
Shouldn't this be `let _url = url`?

Now when someone would do following, it'd request the same url that user has visited instead of api which may break existing stuff. 
```js
const { execute } = useAxios('/api/...', {}, { immediate: false })
execute()
```

Reply:
Failed to consider this business scenario